go/src/staleFlagCleaner.go

```diff
@@ -0,0 +1,624 @@
+/*
+Copyright (c) 2021 Uber Technologies, Inc.
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+
+except in compliance with the License. You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the
+
+License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+express or implied. See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package src
+
+import (
+	"encoding/json"
+	"go/token"
+	"io/ioutil"
+	"log"
+	"os"
+
+	/*
+		We have used dst package instead of official go ast package because ast package
+		does not keep positions of comment in place. So, we use dst package which keeps
+		the positions of comment in place. It is similiar to ast package and have same
+		functions as ast. So you can use documentation of ast to see the details of helper
+		functions.
+	*/
+	"github.com/dave/dst"
+	"github.com/dave/dst/dstutil"
+)
+
+// API : For the type of API
+type API int
+
+const (
+	isTreated API = iota
+	isControl
+	isTesting
+	isUnknown
+)
+
+// Methods : For json it is
+type Methods struct {
+	Methods []FlagAPI `json:"methodProperties"`
+}
+
+// FlagAPI : for getting the type from config file
+type FlagAPI struct {
+	MethodName string `json:"methodName"`
+	OfType     string `json:"flagType"`
+	FlagIndex  int    `json:"argumentIndex"`
+}
+
+// Value : This is made to not confuse with true and false in code
+type Value int
+
+const (
+	isTrue Value = iota
+	isFalse
+	// This will be returned when you don't have to find true or false
+	isBot
+)
+
+// Operator : To get the boolean operators
+type Operator int
+
+const (
+	and Operator = iota
+	or
+	not
+)
+
+// Some useful global variable
+var treated, control string = "treated", "control"
+var strTrue, strFalse string = "true", "false"
```

Comment:
I prefixed with "str" just for kind of reminder that `true` that we are getting in the parsing is of the form of string. So that it doesn't get confused with a boolean one.

Reply:
Changing treated -> modeTreated, control -> modeControl.